homeassistant/components/canary/alarm_control_panel.py

```diff
@@ -53,6 +53,7 @@ class CanaryAlarm(
         | AlarmControlPanelEntityFeature.ARM_AWAY
         | AlarmControlPanelEntityFeature.ARM_NIGHT
     )
+    _attr_code_arm_required = False
```

Comment:
Why is this changed?

Reply:
These integrations doesn't require a code for their methods but was by not implementing `code_arm_required` actually setting it to `True`.
We are with this PR introducing validation for code so without setting this property to `False` these integrations would require a code input from the user.
See blog post link in the PR description.